### PR TITLE
Remove some invalid grants from the import

### PR DESCRIPTION
**Pull request by mattandrews**
Tweaks the data import logic to remove a couple of specific grants which shouldn't be there. Longer term we need a better fix for this, but for now it's workable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ This service is a lambda function which queries an external MongoDB instance for
 
 You'll need to download at least one of the existing datasets from the [GrantNav publisher page](http://grantnav.threesixtygiving.org/publisher/360G-blf) for the Fund.
 
+Make sure the CSV doesn't have a space as the very first character as this has been known to break imports in mysterious ways
```
Comment (davidrapson) [inline README.md]: Any more context on this? Can we make the code more defensive to account for this?

Comment (mattandrews) [reply README.md]: Hm, possibly? Could be a legacy of exporting from MS Excel – only way we could really fix this is test for the first character being a space. It's enough of an edge case that I'm happy to keep it a manual step for now, especially if the import process is likely to change in the near future.
